src/core/Menu/MenuDivider.tsx

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/menu.css';
+
+export const MenuDivider = () => {
+  useTheme();
+  return <li role='separator' className='iui-menu-divider' />;
```

Comment:
Why do you need this role='separator' here?

Reply:
`role='separator'` helps for screen readers to know that this element is not an actual item but a separator and therefore it is being skipped by them. More info in this GitHub issue: https://github.com/Heydon/REVENGE.CSS/issues/12#issuecomment-59908475